appveyor.yml

```diff
@@ -53,6 +53,7 @@ install:
     - cmd: conda config --add channels http://conda.binstar.org/conda-forge
     - cmd: conda info
     - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install conda-build=1.20.0 --yes
```

Comment:
Yeah, it's confusing for sure. There were two AFAICT unrelated bugs.

> Not sure about that. I am fixing the new feedstocks only and this is fixed in staged recipes. So let's hope that a new conda-build is out soon to avoid modifying `conda-smithy`.

I don't think this is a universal problem. There are some specific cases where it occurred, but again this was merely a thought of how to fix before leaving. So, haven't had time to grok everything that has happened with it yet.